src/mlpack/tests/metric_test.cpp

```diff
@@ -40,17 +38,17 @@ BOOST_AUTO_TEST_CASE(L1MetricTest)
 
   ManhattanDistance lMetric;
 
-  BOOST_REQUIRE_CLOSE((double) arma::accu(arma::abs(a1 - b1)),
-                      lMetric.Evaluate(a1, b1), 1e-5);
+  REQUIRE((double) arma::accu(arma::abs(a1 - b1)) ==
+          Approx(lMetric.Evaluate(a1, b1)).epsilon(1e-7));
 
-  BOOST_REQUIRE_CLOSE((double) arma::accu(arma::abs(a2 - b2)),
-                      lMetric.Evaluate(a2, b2), 1e-5);
+  REQUIRE((double) arma::accu(arma::abs(a2 - b2)) ==
+          Approx(lMetric.Evaluate(a2, b2)).epsilon(1e-7));
```

Comment:
```suggestion
      Approx(lMetric.Evaluate(a2, b2)).epsilon(1e-7));
```

See https://github.com/mlpack/mlpack/wiki/DesignGuidelines#tabbing for more information. There are other places where we should apply the patch as well.